scripts/mk-validator-proof.js

```diff
@@ -0,0 +1,91 @@
+/**
```

Comment:
for the future: ask @alanshaw about `access/grant` -- to get out of the habit of non-expiring delegations